### PR TITLE
fix crash when no camera render EidtBox, VIdeoPlayer, WebView on native platform

### PR DESCRIPTION
**Pull request by PPpro**
re: https://github.com/cocos-creator/engine/pull/6969

### CODE REVIEW

**Comment by PPpro** [comment]
@holycanvas

**Comment by PPpro** [comment]
@yongminghuang migrate this bug fix into the runtime adapter repo

File: engine/jsb-webview.js [hunk 1]
```diff
@@ -269,6 +269,9 @@
         this._h = node._contentSize.height;
 
         let camera = cc.Camera.findCamera(node)._camera;
+        if (!camera) {
+            return;
```
Comment (jareguo) [inline engine/jsb-webview.js:273]: Should validate camera at the beginning, just after line 253

Comment (PPpro) [reply engine/jsb-webview.js]: updated

**Review by holycanvas — APPROVED** [review]

**Pull request merged**